tools/vera++/scripts/rules/jerry_switch_case.tcl

```diff
@@ -163,7 +165,10 @@ foreach fileName [getSourceFileNames] {
                             set state "fallthru"
                             set line_num [lindex $next_token 1]
                             continue
-                        } elseif {$next_token_type == "break" || $next_token_type == "return" || $next_token_type == "goto"} {
+                        } elseif {$next_token_type == "break"
+                                  || $next_token_type == "continue"
+                                  || $next_token_type == "return"
+                                  || $next_token_type == "goto"} {
```

Comment:
There is `goto` in several places in the current code. This patch is about only the vera++ check. Please open a separate issue for eliminating `goto` and suggest your solution. I also don't like `goto` too much, but it can be handy in some cases. :)